cli/node/forms.py

```diff
@@ -20,3 +20,11 @@ class GetNodePeersForm(Schema):
     """
 
     node_url = NodeUrlField(required=False)
+
+
+class GetNodeInformationForm(Schema):
+    """
+    Get the node information form.
```

Comment:
`Get information about synchronization and peer count of the node form.`

Reply:
@Alladin9393, done.